### ISSUE TITLE
Defining a default destination URL in the config file

### ISSUE THREAD

**Issue by erikgeurts** [revive-adserver/revive-adserver#1147]
There is a line in the config file to define the URL of a default banner, which will be served in case a request is made for a zone that does not exist.

```
[defaultBanner]
imageUrl=
```

I would like to propose adding another line, which can be used to define the destinationURL for that default banner. Something like:

```
[defaultBanner]
imageUrl=https://www.example.com/banner.jpg
destinationUrl=https://www.example.com/landingpage.html

```

**Comment by mbeccati** [comment]
I think we should review the "feature" in its entirety, as it currently fills all the blanks.

**Comment by andrewatfornax** [comment]
I agree - the feature should be deprecated. Having a global default banner option doesn't make any sense anymore; there are too many different banners sizes, etc. and there are plenty of options for linking/chaining to take care of dealing with cases where a banner is not able to be selected in the initial zone.

**Comment by erikgeurts** [comment]
This has been addressed, and in my view resolved, in #1192. The response that can be configured for a request that references an invalid zone ID can be defined to include any sort of HTML, including if so desired an image URL and a landing page URL.

I propose we move this issue to milestone v5.1.0, mark it as duplicate and resolved, and close it.

**Comment by mbeccati** [comment]
@erikgeurts sure. We should also mention somewhere that using a default image URL is deprecated, so that we can remove it next.
